# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/procps/hotfix-utmp.patch.musl
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- procps-ng-4.0.3/src/w.c.vanilla	2023-06-05 10:23:31.980880966 +0200
+++ procps-ng-4.0.3/src/w.c	2023-06-05 10:24:09.208880517 +0200
@@ -48,9 +48,8 @@
 #include <unistd.h>
 #ifdef HAVE_UTMPX_H
 #	include <utmpx.h>
-#else
-#	include <utmp.h>
 #endif
+#	include <utmp.h>
 #include <arpa/inet.h>
 
 #include "c.h"
